### PR TITLE
update boilerplate

### PR DESCRIPTION
**Pull request by JaskRendix**
Related to #1421 
New boilerplate based on the proposal.
If something it's wrong or missing, then feel free to say it.

### CODE REVIEW

File: run_tuxemon.py [hunk 1]
```diff
@@ -1,31 +1,5 @@
-#!/usr/bin/env python3
```
Comment (bitcraft) [inline run_tuxemon.py:1]: don't drop this line

**Review by bitcraft — COMMENTED** [review]
no changes requested, but we may consider keeping the "contributor" lines.  not sure right now.

File: setup.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env python3
```
Comment (NeonFlames) [inline setup.py:1]: Probably shouldn't drop this

File: run_tuxemon.py (hunk 1 above)
Comment (JaskRendix) [reply run_tuxemon.py:1]: ok

File: setup.py (hunk 2 above)
Comment (JaskRendix) [reply setup.py:1]: ok

**Comment by JaskRendix** [comment]
I can insert again the contributors, no problem, just tell me if you want to be inserted again (you're in 84 files).
I don't know if it's necessary to state it, but I'm not interested in having my name inside the file (created or edited/contributed).

**Review by bitcraft — APPROVED** [review]
looks good, thank you.  ill follow up with the other part of the issue, migrating the contributor lines and the ATTRIBUTIONS file

**Comment by JaskRendix** [comment]
Ok, no problem, in that case this can be useful: [contributors.txt](https://github.com/Tuxemon/Tuxemon/files/10338876/contributors.txt) (it was my backup if I needed to insert again the contributors).

**Review by NeonFlames — APPROVED** [review]

**Review by kerizane — APPROVED** [review]

**Pull request merged**